supervisor/snapshots/__init__.py

```diff
@@ -234,6 +235,10 @@ async def do_restore_full(self, snapshot, password=None):
                     self.sys_homeassistant.core.update(snapshot.homeassistant_version)
                 )
 
+                # Restore docker config
+                _LOGGER.info("Restore %s run Docker Config", snapshot.slug)
+                await snapshot.restore_dockerconfig()
```

Comment:
Since we support custom image for homeassistant, move this to before the homeassistant restore

Reply:
restore_dockerconfig is not awaitable